src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -595,19 +595,27 @@ private static CrontabSchedule VerifyBackupFrequency(string backupFrequency)
 
                 var restorePoints = new RestorePoints();
 
+                if (Directory.Exists(restorePathJson.Path) == false)
+                    throw new InvalidOperationException($"Path '{restorePathJson.Path}' doesn't exist");
```

Comment:
And what if this is a permission problem?

Reply:
fixed